src/commands/new.ts

```diff
@@ -180,19 +399,27 @@ export default {
     )
 
     stopSpinner(" Writing your app name in the sand", "🏝")
+    // #endregion
 
+    // #region Install CocoaPods
     // install pods
-    startSpinner("Baking CocoaPods")
-    await spawnProgress(`npx pod-install@${deps.podInstall}`, {
-      onProgress: log,
-    })
-    stopSpinner("Baking CocoaPods", "☕️")
+    if (installDeps === true) {
+      startSpinner("Baking CocoaPods")
+      await spawnProgress(`npx pod-install@${deps.podInstall}`, {
+        onProgress: log,
+      })
+      stopSpinner("Baking CocoaPods", "☕️")
+    }
```

Comment:
This throws an error if we don't run packager install before, so we don't call this unless we have installed packager dependencies

Reply:
💯  I definitely include this in "deps", so this is appropriate.